docker-compose.override.yml

```diff
@@ -8,117 +8,56 @@ volumes:
   smtp_maildir:
 
 services:
-  traefik-me-certificate-downloader:
-    image: alpine
-    depends_on:
-      envsubst:
-        condition: service_completed_successfully
-    command: sh -c "cd /etc/ssl/traefik
-      && wget traefik.me/fullchain.pem -O traefik.me.crt
-      && wget traefik.me/privkey.pem -O traefik.me-key.pem"
-    volumes:
-      - ./resources/ssl:/etc/ssl/traefik
-
-  georchestra-127-0-1-1.traefik.me:
-    image: traefik:2.9
-    depends_on:
-      traefik-me-certificate-downloader:
-        condition: service_completed_successfully
-    healthcheck:
-      test: traefik healthcheck --ping
+  caddy:
+    image: lucaslorentz/caddy-docker-proxy:2.8-alpine
```

Comment:
no official image ?

Reply:
there is one, but the official image doesn't support docker labels. it's a separate project https://github.com/lucaslorentz/caddy-docker-proxy, I have explained in my first comment.

this project has existed for almost 6 years now and is here to stay: https://github.com/lucaslorentz/caddy-docker-proxy/releases?page=4. [Matt Holt](https://github.com/mholt) is well aware of the existence of this project and gives all of his support.